dci-agent-setup/roles/setup_dci_agent/tasks/main.yaml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Ensure required variables are defined
+  fail: msg="{{ item }} value missing and is required"
+  when: "{{ item }} is not defined"
+  with_items:
+    - dci_group_name
+    - remotcis
```

Comment:
Not sure if the typo is volunteer or not, but I see it consistent. s/remotcis/remotecis/